components/aws/sagemaker/ground_truth/src/ground_truth.py

```diff
@@ -49,6 +49,8 @@ def create_parser():
   parser.add_argument('--max_concurrent_tasks', type=int, required=False, help='The maximum number of data objects that can be labeled by human workers at the same time.', default=0)
   parser.add_argument('--workforce_task_price', type=float, required=False, help='The price that you pay for each task performed by a public worker in USD. Specify to the tenth fractions of a cent. Format as "0.000".', default=0.000)
   parser.add_argument('--tags', type=_utils.yaml_or_json_str, required=False, help='An array of key-value pairs, to categorize AWS resources.', default={})
+  parser.add_argument('--output_manifest_location_output_path', type=str, default='/tmp/manifest-location.json', help='Local output path for the file containing the Amazon S3 bucket location of the manifest file for labeled data.')
```

Comment:
1. file names have changed (active_learning_model_arn -> active-model-arn) and
Q: Does this break backward compatibility? as in the name of GT component output changes? If it does we should avoid breaking change

2. file type changed to json (txt -> json) but in utils the `json_encode` is set to `False`
Q: do some of these need to be txt?

Reply:
KFP overrides all the default values, full file paths and extension. So really these are just placeholders in case someone wants to run the scripts manually, I suppose. I might remove the extensions just to prevent confusion, but in reality it doesn't matter.

Reply:
ok for file name, what about name of output?

Reply:
got it, referenced in component's yaml